Office365-Admin/add-users/get-access-to-and-back-up-a-former-user-s-data.md

```diff
@@ -29,62 +29,59 @@ description: "Learn how to preserve an employee's files and emails when the pers
 
 # Get access to and back up a former user's data
 
- 
+ ::: moniker range="o365-worldwide"
+
+> [!TIP]
+> Need help with the steps in this topic? We’ve got you covered. Make an appointment at your local Microsoft Store with an Answer Desk expert to help resolve your issue. Go to the [Microsoft Stores page](https://go.microsoft.com/fwlink/?LinkID=2041482) and choose your location to schedule an appointment.
+
+::: moniker-end
 
 When an employee leaves your organization, you probably want to access their data (documents and emails) and either review it, back it up, or transfer ownership to a new employee.
   
     
 ## Access a former user's OneDrive documents
 
-
-If you remove a user's Office 365 license but don't delete the account, you retain access to the content in the user's OneDrive site. If you delete their Office 365 account you have 30 days to access a former user’s OneDrive data. If you don't [restore a user account](https://docs.microsoft.com/office365/admin/add-users/restore-user) within 30 days their OneDrive content is deleted. Before you delete the account, you should move the content from their OneDrive to another location.
+If you remove a user's Microsoft 365 license but don't delete the account, you retain access to the content in the user's OneDrive site. If you delete their Microsoft 365 account you have 30 days to access a former user’s OneDrive data. If you don't [restore a user account](https://docs.microsoft.com/office365/admin/add-users/restore-user) within 30 days their OneDrive content is deleted. Before you delete the account, you should move the content from their OneDrive to another location.
 
 To preserve a former user's OneDrive for Business documents you first access their OneDrive site and then move the files. 
 
-1. Sign in to Office 365 with your work or school account. 
-    
-2. Go to the [![Office 365 admin center.](../media/e00ba917-c3fb-4173-b344-43eb5c7eeb15.png)](https://portal.office.com/adminportal/home).
-    
-3. Go to **Active Users** and select the user.
-    
-4. Expand **OneDrive Settings** in the user details pane, and select **Access files**.
-    
-5. Copy the files to your own OneDrive for Business or a common location.
-    
-    There are a few ways to copy files in Office 365. See [Video: Set up document storage and sharing in Office 365](https://support.office.com/article/6a41fb61-9c11-4adf-9b98-f7910fe1b0d2.aspx). Or, to sync your files, and then upload those files to your OneDrive for Business or your team site, see [Sync files with the new OneDrive sync client in Windows](https://support.office.com/article/615391c4-2bd3-4aae-a42a-858262e42a49.aspx).
+1. In the admin center, go to the **User management > [Active users](https://go.microsoft.com/fwlink/p/?linkid=834822)** page.
```

Comment:
You're right, this should not be "User management", just "Users." Also, formatting should be like this so that the > in between the UI is not bold:
… go to the **Users** > [Active users](https:...) page.